### PR TITLE
Contrast module now uses changePixel Method

### LINKED ISSUE #1472: Contrast Module should use changePixel Method
### Please describe the problem (or idea)
The Contrast module currently uses `extraManipulation` but the manipulation process is repetitive so it could use `changePixel` instead which would also WebAssembly accelerate it

Current Code:
https://github.com/publiclab/image-sequencer/blob/12c422a07b55233ae94e13fc3700da2101916c65/src/modules/Contrast/Contrast.js#L5-L25

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by blurry-x-face**
Fixes #1472 

Contrast module now uses changePixel Method instead of extraManipulation which has made it faster.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below
* [x] Insert-step functionality is working correct as expected.
> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
Please make sure to get at least two reviews before asking for merging the PR as that would make the PR more reliable on our part
Thanks!

### CODE REVIEW

**Comment by blurry-x-face** [comment]
@publiclab/is-reviewers please review this

File: src/modules/Contrast/Module.js [hunk 1]
```diff
@@ -15,9 +15,27 @@ module.exports = function Contrast(options, UI) {
...
+    if (contrast > 100) contrast = 100;
+    contrast = (100.0 + contrast) / 100.0;
+    contrast *= contrast;
+
+    function changeContrast(p){
+      p -= 0.5;
+      p *= contrast;
+      p += 0.5;
+      p *= 255;
+      if (p < 0) p = 0;
```
Comment (HarshKhandeparkar) [inline src/modules/Contrast/Module.js]: How about using `Math.min(0, p)` instead of the if stmt?

Comment (blurry-x-face) [reply src/modules/Contrast/Module.js]: Sure

**Comment by jywarren** [comment]
Great work!!

**Pull request merged**